recipes/finemap/meta.yaml

```
@@ -1,37 +1,44 @@
-# This file created by conda-build 3.22.0
-# meta.yaml template originally from:
-# /home/conda/conda-forge/recipes/finemap, last modified Thu Oct 20 22:53:08 2022
-# ------------------------------------------------
+{% set name = "finemap" %}
+{% set version = "1.4.2" %}
 
 package:
-  name: finemap
-  version: 1.4.1
+  name: {{ name }}
+  version: {{ version }}
+
 source:
-  sha256: 75c919d9cc981bc08e1a982b1632110109ffe6a057e803c9aa36c143aa56d546
-  url: http://www.christianbenner.com/finemap_v1.4.1_x86_64.tgz
+  url: http://www.christianbenner.com/finemap_v{{ version }}_x86_64.tgz # [linux]
```

```suggestion
  - url: http://www.christianbenner.com/finemap_v{{ version }}_x86_64.tgz # [linux]
```
This looks wrong. The two pairs (url + sha256) should be two items of YAML list